### ISSUE TITLE
[WAITING ON CHECKLIST] [$1000] Web -  missing translations on additional details page in wallet enablement flow

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#29712]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.84.9
**Reproducible in staging?:** n/a
**Reproducible in production?:** n/a
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @kevinksullivan 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1697480656854849

## Action Performed:
1. User A: login, navigate to Settings > Wallet
2. User A: add personal bank account
3. User B: login, select Request money
4. User B: enter User A as the person you're requesting from
5. User A: Navigate to IOU from User B
5. User A: select Pay with Expensify

## Expected Result:
There should be translations for every field.
- Zip code: código postal
- Personal address: Dirección personal

## Actual Result:
Personal address and zip code both show "MISSING TRANSLATION" on expensify.com emails, to call out that we are missing a translation (https://github.com/Expensify/App/issues/27759 for more details)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01df8c2e9cdd1c685b</li>
        <li>Upwork Job ID: 1714058361895460864</li>
        <li>Last Price Increase: 2024-01-23</li>
<li>Automatic offers: </li>
<ul>
<li>0xmiroslav | Reviewer | 27267086</li>
<li>tienifr | Contributor | 27267089</li>
<li>situchan | Contributor | 28118287</li>
</ul></li>
    </ul>
</details>

**Comment by kevinksullivan** [comment]
looking for translation 

https://expensify.slack.com/archives/C21FRDWCV/p1697498156628909

**Comment by kevinksullivan** [comment]
Updated the OP. the problem is that we are missing translations, and not that MISSING TRANSLATION is shown on expensify accounts.

**Comment by mountiny** [comment]
@kevinksullivan as @ishpaul777 mentioned, the problem is that we provide already translated text to the translate method again, we should make sure we are not passing this through the translation twice

**Comment by mountiny** [comment]
@tienifr @0xmiroslav This has popped up in couple of flows in the app, for example here too https://github.com/Expensify/App/pull/28907 Could we please fix all these occurences here and try to fix this as generically as possible for Form component? thanks!

**Comment by marcaaron** [comment]
honestly i don't understand how this is a new issue and not a regression caused by that PR but 🤷‍♂️

**Comment by mountiny** [comment]
@marcaaron I can take it over if you prefer.

I dont think its a regression because that PR just surfaced this bad practice which should be fixed imho.

**Comment by marcaaron** [comment]
It _looks like a bug_. I thought all the server errors show `MISSING TRANSLATION`? Why do we want this? 😂

**Comment by mallenexpensify** [comment]
PR is taking a LONG time, removed `Reviewing` here so it'll go overdue, bumped to weekly too.  @tienifr can you provide an update? (also commented in the PR).  Thx

**Comment by quinthar** [comment]
Why is this taking so long?

**Comment by quinthar** [comment]
Can we please reassign to a new contribuotr if @tienifr is unable to finish this immediately?

**Comment by peterdbarkerUK** [comment]
@0xmiroslav I think [this is waiting on review](https://github.com/Expensify/App/pull/30073) - could you take a look ASAP?

**Comment by mallenexpensify** [comment]
@peterdbarkerUK , per this chat in #contributor-plus, I've reassigned to @situchan 
https://expensify.slack.com/archives/C02NK2DQWUX/p1706043173197049

**Comment by jliexpensify** [comment]
Guessing we're also putting this on hold @mallenexpensify , in favour of the `Missing Translation` PR?

**Comment by jliexpensify** [comment]
Not overdue, @tienifr is working on the PR

**Comment by situchan** [comment]
Not overdue

**Comment by mallenexpensify** [comment]
[PR was merged yesterday](https://github.com/Expensify/App/pull/30073#event-11750595222)!!!

**Comment by situchan** [comment]
Not overdue

**Comment by jliexpensify** [comment]
Bump @situchan to complete the checklist please

**Comment by jliexpensify** [comment]
I made a new job + here is a payment summary:

- Contributor: @tienifr $1000
- C+: @situchan $1000

[Upworks job](https://www.upwork.com/jobs/~018209491a0bdfe81d)

**Comment by situchan** [comment]
I already have offer accepted [here](https://github.com/Expensify/App/issues/29712#issuecomment-1906912453)

This doesn't require BZ checklist as no way to reproduce on non-Expensify accounts.

**Comment by jliexpensify** [comment]
@situchan I'm aware that you've accepted the offer, but that was for the old posting and it's not been closed by Upworks:

![image](https://github.com/Expensify/App/assets/30815269/f3a5788f-5dba-4148-8c4c-c71d6cd24dc5)

I don't have a way to pay you on that job, so I created a new, active job.

**Comment by situchan** [comment]
ok no problem. Accepted new offer

**Comment by jliexpensify** [comment]
Paid and job closed. Waiting on @situchan to complete checklist

**Comment by situchan** [comment]
@jliexpensify here's my feedback on checklist - https://github.com/Expensify/App/issues/29712#issuecomment-1951680561

**Comment by jliexpensify** [comment]
Cool, thanks! Closing!
